packages/viewer/src/components/viewer/viewer.tsx

```diff
@@ -201,14 +206,30 @@ export class Viewer {
     }
 
     if (this.cameraControls) {
-      this.registerInteractionHandler(new MouseInteractionHandler());
-      this.registerInteractionHandler(new TouchInteractionHandler());
+      // default to pointer events if allowed by browser.
+      if (this.usePointerEvents) {
```

Comment:
Could probably just inline `window.PointerEvent != null` here. The additional property isn't adding any value.